packages/server/package.json

```diff
@@ -56,11 +56,11 @@
     "jsdom": "^15.1.1",
     "json-loader": "^0.5.7",
     "lint-staged": "^10.1.7",
-    "mocha": "^5.2.0",
+    "mocha": "^9.0.3",
     "mockdate": "^2.0.1",
     "prettier": "^1.19.1",
-    "relay-compiler": "^8.0.0",
-    "relay-compiler-language-typescript": "^13.0.2",
+    "relay-compiler": "^11.0.2",
```

Comment:
oooo this might break stuff. i'll have to test to make sure everything compiles correctly still

Reply:
Everything seem'ed to still work as long as I didn't also upgrade relay-config. (I know we talked about this yesterday, but wanted to add it to the comments)

Reply:
Disregard, you were right it did break when I upgraded relay-compiler.

Reply:
I upgraded all relay-compiler, to include relay-config and relay-runtime and this resolved the issues.